apps/lib/blockly/tg_tj.js

```diff
@@ -18,14 +18,14 @@ Blockly.Msg.COLOUR_PICKER_TOOLTIP = "Choose a color from the palette.";
 Blockly.Msg.COLOUR_RANDOM_HELPURL = "http://randomcolour.com";
 Blockly.Msg.COLOUR_RANDOM_TITLE = "random color";
 Blockly.Msg.COLOUR_RANDOM_TOOLTIP = "Choose a color at random.";
-Blockly.Msg.COLOUR_RGB_BLUE = "кабуд";
-Blockly.Msg.COLOUR_RGB_GREEN = "сабз";
+Blockly.Msg.COLOUR_RGB_BLUE = "blue";
+Blockly.Msg.COLOUR_RGB_GREEN = "green";
 Blockly.Msg.COLOUR_RGB_HELPURL = "http://www.december.com/html/spec/colorper.html";
-Blockly.Msg.COLOUR_RGB_RED = "сурх";
+Blockly.Msg.COLOUR_RGB_RED = "red";
 Blockly.Msg.COLOUR_RGB_TITLE = "color with";
 Blockly.Msg.COLOUR_RGB_TOOLTIP = "Create a color with the specified amount of red, green, and blue.  All values must be between 0 and 100.";
-Blockly.Msg.CONFIRM_CREATE_VARIABLE = "Сохтан";
-Blockly.Msg.CONFIRM_DELETE_FUNCTION_MESSAGE = "Огоҳӣ! Дар ҳақиқат мехоҳед %1 -ро кӯр кунед?";
+Blockly.Msg.CONFIRM_CREATE_VARIABLE = "Create";
+Blockly.Msg.CONFIRM_DELETE_FUNCTION_MESSAGE = "Warning! Are you sure you want to delete %1?";
```

Comment:
Yeah... very strange. I wonder if you download the file straight from Crowdin, you should be able to confirm that it downloads with the translations. In which case it's probably something with our out script and where we merge/use fallbacks?